mne/io/brainvision/brainvision.py

```diff
@@ -151,8 +151,9 @@ def _read_vmrk(fname):
         The onsets in seconds.
     description : array, shape (n_annots,)
         The description of each annotation.
-    orig_time : str
-        The origin time as a string.
+    date_str : str
```

Comment:
why do we change the return name? I know that during the reading is called `date_str` but what we are returning is `orig_time`. where's the problem? is it called `orig_time` or `date_str` afterwards?

Reply:
I mean both works.

Reply:
we are not returning `orig_time`, we are returning a `date_str` of the format `YYYYMMDDHHMMSSuuuuuu` ... note: prior to this PR, this could also be `None`, which is not a good idea because we will be passing this to `_str_to_meas_date` (see below)

I think `date_str` is more transparent about what we actually read and return